packages/amplify-category-geo/src/__tests__/service-stacks/__snapshots__/mapStack.test.ts.snap

```diff
@@ -112,10 +112,6 @@ Object {
       "UpdateReplacePolicy": "Delete",
     },
     "CustomMapLambda51D5D430": Object {
-      "DependsOn": Array [
-        "CustomMapLambdaServiceRoleDefaultPolicy983FDD4E",
-        "CustomMapLambdaServiceRole4EE7732C",
-      ],
       "Properties": Object {
```

Comment:
in data repo those kind of deletion was first sign of later problems. 
Do we know why this disappeared?

Reply:
This is happening because I removed prepareApp method. I added similar functionality (similar code),it added these dependsOn  but since the default synthesizer is changed for cdkV2 , its adding an extra bootstrap cfn parameter.

Reply:
does this mean we're good with this?

Reply:
I am not sure , I have to check by running some e2es.  I'll create a task for this to investigate. 
to which asana project I can add this task ?

Reply:
CDK v2 project.

Reply:
https://app.asana.com/0/1202389680978480/list